libparsec/src/bin/cli.rs

```diff
@@ -0,0 +1,30 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+use clap::{Parser, Subcommand};
+use std::path::PathBuf;
+
+use libparsec::get_default_config_dir;
+
+/// Parsec cli
+#[derive(Parser)]
+struct Arg {
+    #[command(subcommand)]
+    command: Command,
+}
+
+#[derive(Subcommand)]
+enum Command {
+    /// List all devices
+    ListDevices { config_dir: Option<PathBuf> },
```

Comment:
use a struct for the argument of `ListDevices`, that allow to define the struct on `cli::list_devices`

Reply:
Done